src/client/content_mapblock.cpp

```diff
@@ -46,30 +46,30 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 static const v3s16 light_dirs[8] = {
 	v3s16(-1, -1, -1),
-	v3s16(-1, -1,  1),
-	v3s16(-1,  1, -1),
-	v3s16(-1,  1,  1),
-	v3s16( 1, -1, -1),
-	v3s16( 1, -1,  1),
-	v3s16( 1,  1, -1),
-	v3s16( 1,  1,  1),
+	v3s16(-1, -1, 1),
+	v3s16(-1, 1, -1),
+	v3s16(-1, 1, 1),
+	v3s16(1, -1, -1),
+	v3s16(1, -1, 1),
+	v3s16(1, 1, -1),
+	v3s16(1, 1, 1),
```

Comment:
Is it possible to make clang-format ignore whitespaces used for alignment? Preferably without `clang-format off`. :P

Reply:
Unfortunately i cannot see how an auto-formatter can possibly recognise and preserve whitespace for alignment. CF managed to do this is in one place in mapgen files, but generally does not.
I think we will need to use CF on/off lines a lot.